bitnami/metrics-server/templates/serviceaccount.yaml

```diff
@@ -4,4 +4,7 @@ kind: ServiceAccount
 metadata:
   name: {{ template "common.names.fullname" . }}
   labels: {{- include "common.labels.standard" . | nindent 4 }}
+{{- if .Values.serviceAccount.automountServiceAccountToken }}
```

Comment:
Hi @migruiz4 and @joshuastern,
This change does not make sense anymore ;-)
When you provide `false` in the values, this conditional does not render the `false`

Reply:
Hi @mkilchhofer,

My bad, thank you for raising this issue.